packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -42,9 +42,9 @@ extension InspectorColorScheme on ColorScheme {
       isLight ? Colors.yellowAccent : const Color.fromARGB(255, 70, 73, 76);
 }
 
-const double iconPadding = 5.0;
+const double iconPadding = 4.0;
 const double chartLineStrokeWidth = 1.0;
-double get columnWidth => isDense() ? 12 : 16.0;
+const double columnWidth = 16.0;
```

Comment:
why is this removed? We use dense mode for embedding the inspector in the IDEs.

Reply:
sorry, put it back